src/drtvam/geometry.py

```diff
@@ -95,6 +95,49 @@ def to_dict(self):
         d = self.add_occlusions(d)
         return d
 
+class CustomVial(Container):
+    def __init__(self, params):
+        super().__init__(params)
+
+        if "filename_vial_outer" not in params.keys() or "filename_vial_inner" not in params.keys():
+            raise ValueError(f"[{self.__class__.__name__}] Missing fields 'filename_vial_outer' or 'filename_vial_inner' for custom vial.")
+
+        self.vial_ior = params['ior']
+        # it is important that both surface have the normals pointing in the
+        # right directions
+        self.filename_vial_outer = params["filename_vial_outer"]
+        self.filename_vial_inner = params["filename_vial_inner"]
+
+    def to_dict(self):
+        d = {
+            'printing_medium' : self.medium_dict(),
+            'vial_exterior' : {
+                'type': 'ply',
+                "face_normals": True,
```

Comment:
@bathal1 if i don't set face_normals it somehow makes the object round. Like some sort of inter node interpolation?